src/config/formSources.config.js

```diff
@@ -4,11 +4,13 @@ export const formSources = {
       key: 'suffolkListLab',
       url: 'https://apps.suffolklitlab.org',
       name: 'Suffolk LIT Lab',
+      path: 'ma',
     },
     {
       key: 'greaterBostonLegalService',
       url: 'https://interviews.gbls.org',
       name: 'Greater Boston Legal Services',
+      path: 'gb',
```

Comment:
Just wanted to double-check: it's possible for two servers to share the same path, right?

Reply:
What do you envision determining which servers are associated with a given path, should that be a seperate config where each path has an array of servers they pull from?

Reply:
It's important now that multiple servers can point to the same path, since we're thinking of the path as relating to the jurisdiction. The two example servers we're pulling from here are both in Massachusetts and would be syndicated to the same path. 

In the future, we'd like to consider hosting a small number of forms for outside jurisdictions. For example: if we help Minnesota build 3 forms, it would be nice if those could go to courtformsonline.org/mn even if they're hosted on apps.suffolklitlab.org. That would be something we'd just do for relatively small numbers of forms.

I can also see multiple servers around the country having US-national level forms that should all go to the same path, /us/.

To get that outcome, I trust your judgment on the proper implementation.

Here's one path that might meet all of our needs for flexibility:

1. Each path is associated with a jurisdiction code or codes, and will show all forms matching that jurisdiction's codes
2. Each interview can specify the jurisdiction code or codes in its metadata (the JSON endpoint)
3. Each server has a default jurisdiction code, which will be used if the interview doesn't specify one

If this feature is getting too complex, happy to talk it through more and figure out the right tradeoffs.

Reply:
Gotcha, thanks for the clarification